app/models/embed/purl/media_file.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module Embed
+  class Purl
+    # Any file on a media object, may be a mp4 or a caption file.
+    class MediaFile < ResourceFile
+      attr_accessor :sdr_generated, :language
+      alias sdr_generated? sdr_generated
```

Comment:
Can we note that sdr_generated is not a media only field, but we are currently only using it for media. I would imagine eventually we are going to want to use this field for PDFs at some point.

Reply:
The representations we are using here are not the same as the representations in the Cocina.  This model is just for "as we use it in the context of sul-embed".  That said, I'd feel comfortable putting a note in the JSON deserializer as that does deal with Cocina.

Reply:
https://github.com/sul-dlss/sul-embed/pull/2683/files#diff-29d922aaac25053d3f87aa4cb3eafedf452ee03342ec6deb21ccd4b962eb6bbfR71